kork/kork-web/src/main/java/com/netflix/spinnaker/okhttp/SpinnakerRequestHeaderInterceptor.java

```diff
@@ -18,31 +18,52 @@
 
 import com.netflix.spinnaker.kork.common.Header;
 import com.netflix.spinnaker.security.AuthenticatedRequest;
+import io.micrometer.core.instrument.util.StringUtils;
 import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
 import okhttp3.Interceptor;
 import okhttp3.Request;
 import okhttp3.Response;
+import org.slf4j.MDC;
 
 /**
  * As {@link retrofit.RequestInterceptor} no longer present in retrofit2, we have to use {@link
  * okhttp3.Interceptor} to add authenticated headers to requests.
  */
+@Slf4j
```

Comment:
https://github.com/spinnaker/spinnaker/blob/8b5c743e650a8d706bb7079147cfda75305cdd93/gate/gate-core/src/main/java/com/netflix/spinnaker/gate/filters/CorsFilter.java#L32 should probably use log4j2